File: app/src/main/java/com/mahesh_prajapati/itunes/utils/cardstack/internal/AnimationSetting.java
package com.mahesh_prajapati.itunes.utils.cardstack.internal;

import android.view.animation.Interpolator;

import com.mahesh_prajapati.itunes.utils.cardstack.Direction;


public interface AnimationSetting {
    Direction getDirection();
    int getDuration();
    Interpolator getInterpolator();
}

File: CardSwapView/src/main/java/com/mahesh_prajapati/cardswapview/internal/AnimationSetting.kt
package com.mahesh_prajapati.cardswapview.internal

import android.graphics.Path
import android.view.animation.Interpolator
import com.mahesh_prajapati.cardswapview.Direction


interface AnimationSetting {
    fun getDirection(): Direction
    fun getDuration(): Int
    fun getInterpolator(): Interpolator
}